examples/common/tracing/TraceDecoder.cpp

```diff
@@ -272,6 +272,13 @@ CHIP_ERROR TraceDecoder::MaybeLogAndConsumePayload(Json::Value & json, bool isRe
     auto size = static_cast<uint16_t>(json[kPayloadSizeKey].asLargestUInt());
     if (size)
     {
+        {
+            auto scoppedIndent = ScopedLogIndentWithSize("Decrypted Payload", size);
+
+            auto payload = json[kPayloadDataKey].asString();
+            Log("data", payload.c_str());
```

Comment:
Payload can have embedded nulls, no?  We need to do whatever we would normally do to log a random bytespan here...

Reply:
No, the payload at this layer is hex

Reply:
Ah @tcarmelveilleux was faster than me.
Basically `kPayloadDataKey` maps to `payload_hex` which is encoded as hex there:
 *  https://github.com/project-chip/connectedhomeip/blob/caacbdef39b2ce69a918979cefd2c5d8448a22e1/examples/common/tracing/TraceHandlers.cpp#L282
 * https://github.com/project-chip/connectedhomeip/blob/caacbdef39b2ce69a918979cefd2c5d8448a22e1/examples/common/tracing/TraceHandlers.cpp#L236
 * https://github.com/project-chip/connectedhomeip/blob/caacbdef39b2ce69a918979cefd2c5d8448a22e1/examples/common/tracing/TraceHandlers.cpp#L258

Reply:
Ah, ok, great.